homeassistant/components/opentherm_gw/climate.py

```diff
@@ -101,6 +102,7 @@ def update_options(self, entry):
     @callback
     def receive_report(self, status):
         """Receive and handle a new report from the Gateway."""
+        self._available = status != {}
```

Comment:
```suggestion
        self._available = bool(status)
```

Reply:
Thanks for the suggestion, I will take care of it tomorrow.